python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -119,32 +119,18 @@ def run(self, inputCat, wcs, calib):
         return outputCat
 
 
-class RunTransformConfig(pexConfig.Config):
-    """!Configuration for RunTransformTask."""
+class RunTransformConfigBase(object):
+    """!Configuration for RunTransformTaskBase derivatives."""
     transform = pexConfig.ConfigurableField(
         doc="Subtask which performs transformations",
         target=TransformTask
     )
-    measConfig = pexConfig.Field(
-        dtype=str,
-        doc="Dataset type of measurement operation configuration",
-        default="processCcd_config"
-    )
-    measType = pexConfig.ChoiceField(
-        dtype=str,
-        doc="Type of measurement operation performed",
-        default="SingleFrame",
-        allowed={
-            "SingleFrame": "Single frame measurement",
-            "Forced": "Forced measurement"
-        }
-    )
 
 
-class RunTransformTask(pipeBase.CmdLineTask):
+class RunTransformTaskBase(object):
```

Comment:
Why do you need to inherit from `object` here and multiply-inherit later in `makeTransformCmdLineTask`?  It seems like you could just inherit from `CmdLineTask`, even if this intermediate base class remains abstract.